numpy/lib/_shape_base_impl.py

```diff
@@ -1185,12 +1185,12 @@ def kron(a, b):
     return result if not is_any_mat else matrix(result, copy=False)
 
 
-def _tile_dispatcher(A, reps):
-    return (A, reps)
+def _tile_dispatcher(A, reps, axis=None):
+    return (A, reps, axis)
```

Comment:
`axis` cannot be an array so should not be passed on (since one cannot override the function based on `axis`).